teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Teacher
+  attr_accessor :specialization
+
+  def initialize(specialization)
+    super(name, age)
+    @specialization = specialization
+  end
```

Comment:
- [ ] Your initialize would should look something like below :+1: to give your teacher a name and age, right :thinking:
```
def initialize(age, specialization, name = "Unknown")
  super(age, name)
  @specialization = specialization
end
```

Reply:
@HAMMAS-SALEEM  thanks
fixed